src/main/java/model/Draw.java

```diff
@@ -0,0 +1,5 @@
+package model;
+
+public interface Draw {
+    void drawing();
```

Comment:
the name of the interface should be a noun  or adjective
and the name of the method should start from the verb